dist/etc/conf/nucleo-wl55jc.env

```diff
@@ -0,0 +1,84 @@
+HIL_DUT_RST_GPIO?=
+HIL_DUT_RST_PORT?=
+HIL_DUT_RST_PIN?=
+
+HIL_DUT_CTS_GPIO?=
+HIL_DUT_CTS_PORT?=
+HIL_DUT_CTS_PIN?=
+
+HIL_DUT_RTS_GPIO?=
+HIL_DUT_RTS_PORT?=
+HIL_DUT_RTS_PIN?=
+
+HIL_DUT_RX_GPIO?="PA2/CN9-7/D1"
+HIL_DUT_RX_PORT?=0
+HIL_DUT_RX_PIN?=2
+
+HIL_DUT_TX_GPIO?="PA3/CN9-8/D0"
+HIL_DUT_TX_PORT?=0
+HIL_DUT_TX_PIN?=3
+
+HIL_DUT_NSS_GPIO?="PA4/CN5-3/D10"
+HIL_DUT_NSS_PORT?=0
+HIL_DUT_NSS_PIN?=4
+
+HIL_DUT_SCK_GPIO?="PA5/CN5-6/D13"
+HIL_DUT_SCK_PORT?=0
+HIL_DUT_SCK_PIN?=5
+
+HIL_DUT_MISO_GPIO?="PA6/CN5-5/D12"
+HIL_DUT_MISO_PORT?=0
+HIL_DUT_MISO_PIN?=6
+
+HIL_DUT_MOSI_GPIO?="PA7/CN5-4/D11"
+HIL_DUT_MOSI_PORT?=0
+HIL_DUT_MOSI_PIN?=7
+
+HIL_DUT_SCL_GPIO?="PA12/CN5-10/D15"
+HIL_DUT_SCL_PORT?=0
+HIL_DUT_SCL_PIN?=12
+
+HIL_DUT_SDA_GPIO?="PA11/CN5-9/D14"
+HIL_DUT_SDA_PORT?=0
+HIL_DUT_SDA_PIN?=11
+
+HIL_DUT_PWM_GPIO?="PB3/CN9-5/D3"
+HIL_DUT_PWM_PORT?=1
+HIL_DUT_PWM_PIN?=3
+
+HIL_DUT_DAC_GPIO?="PB1/CN8-1/A0"
+HIL_DUT_DAC_PORT?=1
+HIL_DUT_DAC_PIN?=1
+
+HIL_DUT_ADC_GPIO?=
+HIL_DUT_ADC_PORT?=
+HIL_DUT_ADC_PIN?=
+
+HIL_DUT_IC_GPIO?="PB8/CN9-3/D5"
+HIL_DUT_IC_PORT?=1
+HIL_DUT_IC_PIN?=8
+
+HIL_DUT_GPIO0_GPIO?="PC2/CN5-1/D8"
+HIL_DUT_GPIO0_PORT?=2
+HIL_DUT_GPIO0_PIN?=2
+
+HIL_DUT_GPIO1_GPIO?="PC1/CN9-1/D7"
+HIL_DUT_GPIO1_PORT?=2
+HIL_DUT_GPIO1_PIN?=1
+
+HIL_DUT_GPIO2_GPIO?="PB5/CN9-4/D4"
```

Comment:
Somehow this is wrong too `B5/CN9-4/D4` the location seems fine maybe it is the PB5?

Reply:
Can you please check this image? I referred to the same. 
![Datasheet](https://user-images.githubusercontent.com/5170658/113739111-c3c18a80-96ff-11eb-8d88-c203fcd7ed6a.png)